schema/test/unit/functions/handle_default_form_change_commit_test.sql

```diff
@@ -171,6 +170,32 @@ select results_eq(
   'All proper values were inserted when form_data includes a key that does not correspond to a column in the target table'
 );
 
+-- function updates the table with null values when the form_data does not include a key that corresponds to a column in the target table
+truncate mock_schema.mock_table restart identity;
+insert into mock_schema.mock_table (text_col, int_col, bool_col, required_col, defaulted_col)
```

Comment:
I think we additionally need tests to make sure it doesn't wipe anything it's not supposed to, e.g. `archived_at`.